ocpp2.0.1/csms.go

```diff
@@ -811,11 +811,15 @@ func (cs *csms) SendRequestAsync(clientId string, request ocpp.Request, callback
 
 func (cs *csms) Start(listenPort int, listenPath string) {
 	// Overriding some protocol-specific values in the lower layers globally
-	ocppj.FormationViolation = ocppj.FormatViolationV2
+	cs.server.Endpoint.Dialect = ocpp.V16
```

Comment:
I guess this should be:

```suggestion
	cs.server.Endpoint.Dialect = ocpp.V2
```